.lintr

```diff
@@ -1,6 +1,6 @@
 linters: with_defaults( # The following TODOs are part of an effort to have {lintr} lint-free (#584)
    line_length_linter = line_length_linter(120),
-   cyclocomp_linter = cyclocomp_linter(29), # TODO reduce to 15
+   cyclocomp_linter = cyclocomp_linter(30), # TODO reduce to 15
```

Comment:
#673 added 1 to the cyclocomp of `lint()`.
To keep this separate from removing ~ 200 lints from tests/, I temporarily bump the limit here.